docs/guides/Create-UWP-Packages-C#.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Create NuGet Packages for the Universal Windows Platform
+description: An end-to-end walkthrough of creating NuGet packages using a Windows Runtime Component for the Universal Windows Platform.
+author: karann-msft
+ms.author: karann
+ms.date: 03/21/2017
+ms.topic: tutorial
+---
+
+# Create UWP packages (C#)
+
+The [Universal Windows Platform (UWP)](https://developer.microsoft.com/windows) provides a common app platform for every device that runs Windows 10. Within this model, UWP apps can call both the WinRT APIs that are common to all devices, and also APIs (including Win32 and .NET) that are specific to the device family on which the app is running.
+
+In this walkthrough you create a NuGet package with a C# UWP component (including a XAML control) that can be used in both Managed and Native projects.
+
+## Prerequisites
+
+1. Visual Studio 2019. Install the 2019 Community edition for free from [visualstudio.com](https://www.visualstudio.com/); you can use the Professional and Enterprise editions as well.
+
+1. NuGet CLI. Download the latest version of `nuget.exe` from [nuget.org/downloads](https://nuget.org/downloads), saving it to a location of your choice (the download is the `.exe` directly). Then add that location to your PATH environment variable if it isn't already.
```

Comment:
I think adding a link to [`Windows specific NuGet CLI doc`](https://docs.microsoft.com/en-us/nuget/reference/nuget-exe-cli-reference#windows) will helps us to avoid repeating the same content.